app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java

```diff
@@ -202,15 +202,11 @@ public String getPageContents(Context applicationContext) {
     private String getTemplatizedCreatedDate() {
         if (dateCreated != null) {
             if (UploadableFile.DateTimeWithSource.EXIF_SOURCE.equals(dateCreatedSource)) {
-                return String.format(Locale.ENGLISH, TEMPLATE_DATE_ACC_TO_EXIF, DateUtil.getDateStringWithSkeletonPattern(dateCreated, "yyyy-MM-dd")) + "\n";
+                java.text.SimpleDateFormat dateFormat = new java.text.SimpleDateFormat("yyyy-MM-dd");
+                return String.format(Locale.ENGLISH, TEMPLATE_DATE_ACC_TO_EXIF, dateFormat.format(dateCreated)) + "\n";
             } else {
-                Calendar calendar = Calendar.getInstance();
-                calendar.setTime(dateCreated);
-                calendar.setTimeZone(TimeZone.getTimeZone("UTC"));
-                return String.format(Locale.ENGLISH, TEMPLATE_DATA_OTHER_SOURCE,
-                        calendar.get(Calendar.YEAR),
-                        calendar.get(Calendar.MONTH),
-                        calendar.get(Calendar.DAY_OF_MONTH)) + "\n";
+                java.text.SimpleDateFormat dateFormat = new java.text.SimpleDateFormat("yyyy-MM-dd");
```

Comment:
dateFormat could be placed out of if-else, the same thing is used in both if and else